.gitignore

```diff
@@ -5,3 +5,4 @@ phpunit.xml
 /Homestead.json
 /after.sh
 /aliases
+/.idea
```

Comment:
This should be in a user's global gitignore file. Projects shouldn't be responsible for ignoring files generated outside of the project.